### PR TITLE
Add testing.

### PR DESCRIPTION
**Pull request by simesy**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: quay.io/govcms/govcms-ci
+    steps:
+      - checkout
+      - run: composer validate --strict || true
```
Comment (AlexSkrypnyk) [inline .circleci/config.yml]: any reason why it is `||true`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      - run:
+          name: Setup codebase
+          command: |
+            git checkout -b latest-package-"${CIRCLE_SHA1}"
+            git clone https://github.com/govCMS/govCMS8-scaffold-paas /app
+            composer --working-dir=/app config repositories.test path "/home/project"
+            composer --working-dir=/app update
+            composer --working-dir=/app require govcms/require-dev:dev-latest-package-"${CIRCLE_SHA1}" --ignore-platform-reqs
+      - run:
+          name: Test settings via drush
```
Comment (AlexSkrypnyk) [inline .circleci/config.yml]: misleading name

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+          working_directory: /app
+          command: |
+            composer info govcms/require-dev
```
Comment (AlexSkrypnyk) [inline .circleci/config.yml]: what does this actually test?

Comment (simesy) [reply .circleci/config.yml]: it just ensures it remains a valid includable composer package.

File: .circleci/config.yml (hunk 2 above)
Comment (simesy) [reply .circleci/config.yml]: I think that got copied from another test.

File: .circleci/config.yml (hunk 1 above)
Comment (simesy) [reply .circleci/config.yml]: Probably just informative. But I think it has come from another test i copied.

File: .circleci/config.yml (hunk 2 above)
Comment (srowlands) [inline .circleci/config.yml:18]:
```suggestion
          name: Ensure require-dev package is retrievable
```

**Pull request merged**